test/system/255-auto-update.bats

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests for automatically update images for containerized services
+#
+
+load helpers
+
+UNIT_DIR="/usr/local/lib/systemd/system"
+
+function setup() {
+    skip_if_remote "systemd tests are meaningless over remote"
+    skip_if_rootless
+
+    basic_setup
+}
+
+function generate_service() {
+    fake_img=$1
+    target_img=$2
+    cmd=$3
+    label=$4
+
+    cname=$(random_string)
+    run_podman tag $fake_img $target_img
+    if [ "$4" ]; then
+        run_podman run -d --name $cname --label "io.containers.autoupdate=$label" $target_img $cmd
+    else
+        run_podman run -d --name $cname $target_img $cmd
+    fi
+
+    run_podman generate systemd --new $cname
+    echo "$output" > "$UNIT_DIR/container-$cname.service"
+    run_podman rm -f $cname
+
+    systemctl daemon-reload
+
+    run systemctl start container-$cname
+    if [ $status -ne 0 ]; then
+        die "Error starting systemd unit $SERVICE_NAME, output: $output"
+    fi
+
+    run systemctl status container-$cname
+    if [ $status -ne 0 ]; then
+        die "Non-zero status of systemd unit $SERVICE_NAME, output: $output"
+    fi
+}
+
+@test "podman auto-update - label io.containers.autoupdate=image" {
```

Comment:
IDK what's up, but this test isn't able to start up systems in CI.

Reply:
Seems my testing env on Fedora 32 has different systemd set up directory. Already update the code to fix this. Thanks for the comments.